content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -29,20 +29,19 @@ The cluster that `kubeadm init` and `kubeadm join` set up should be:
    - lock-down the kubelet API
    - locking down access to the API for system components like the kube-proxy and CoreDNS
    - locking down what a Bootstrap Token can access
-   - etc.
  - **Easy to use**: The user should not have to run anything more than a couple of commands:
    - `kubeadm init`
    - `export KUBECONFIG=/etc/kubernetes/admin.conf`
    - `kubectl apply -f <network-of-choice.yaml>`
    - `kubeadm join --token <token> <master-ip>:<master-port>`
  - **Extendable**:
-   - It should for example _not_ favor any network provider, instead configuring a network is out-of-scope
-   - Should provide the possibility to use a config file for customizing various parameters
+   - It should _not_ favor any particular network provider; instead, configuring a network is out-of-scope
+   - It should provide the possibility to use a config file for customizing various parameters
 
 ## Constants and well-known values and paths
 
-In order to reduce complexity and to simplify development of an on-top-of-kubeadm-implemented deployment solution, kubeadm uses a
-limited set of constants values for well know-known paths and file names.
+In order to reduce complexity and to simplify development od deployment solutions built on top of kubeadm, kubeadm uses a
```

Comment:
`In order to reduce complexity and to simplify development of higher level tools that build on top of kubeadm, it uses a limited set of constants values for well know-known paths and file names.`